<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecTagPage</class>
 <widget class="QWidget" name="SecTagPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_13">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>441</width>
     <height>174</height>
    </rect>
   </property>
   <property name="title">
    <string>主体安全标签</string>
   </property>
   <widget class="QPushButton" name="freshUserSafeTagButton">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>132</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setUserTagButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>391</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_9">
     <item row="2" column="2">
      <widget class="QComboBox" name="u_whole_tagcomboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s15</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>完整性标记：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>安全性标记：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="users_comboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="u_sec_tagcomboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s15</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_12">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>441</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>客体安全标签</string>
   </property>
   <widget class="QPushButton" name="freshFileTagButton">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>140</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setFileTagButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget1">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>391</width>
      <height>105</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_10">
     <item row="0" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>文件名：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>安全性标记：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>完整性标记：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="filenameEdit">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="browserButton">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="isDirheckBox">
       <property name="text">
        <string>目录</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="f_sec_tagcomboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s15</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="f_whole_tagcomboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s15</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
